addons/recolor-custom-blocks/addon.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Recolor custom blocks",
+  "description": "Changes the colors of custom blocks that start with the name of a category followed by a colon (:), for example \"looks: animate\".",
+  "credits": [
+    {
+      "name": "FlamedDogo99",
+      "link": "https://github.com/FlamedDogo99"
+    }
+  ],
+  "tags": ["editor", "codeEditor", "featured"],
+  "relatedAddons": ["data-category-tweaks-v2", "editor-theme3"],
```

Comment:
Could this addon be added to the `relatedAddons` list of the other two as well? Especially data category tweaks - I'm not sure about theme3 because it already has a lot of related addons.